modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -3,6 +3,7 @@
 require 'sidekiq'
 
 module VAForms
+  # rubocop:disable Metrics/MethodLength
```

Comment:
I'd prefer we disable cops via .rubocop_todo.yml instead of in file unless we think shortening the method length here would be wrong (which I don't know when that would ever be the case). Even better, shorten the method via extraction.

Reply:
@ericboehs Sure, I shortened it. Will check it in